src/app/tabs/runTab/settings.js

```diff
@@ -194,6 +196,36 @@ class SettingsUI {
     this.event.trigger('clearInstance', [])
     this.updateNetwork()
     this.fillAccountsList()
+    this.updatePlusButton()
+  }
+
+  updatePlusButton () {
+    // enable/disable + button
+    let plusBtn = document.getElementById('remixRunPlus')
+    let plusTitle = document.getElementById('remixRunPlusWraper')
+    switch (this.selectExEnv.value) {
+      case 'injected': {
+        plusBtn.classList.add(css.disableMouseEvents)
+        plusTitle.title = "Unfortnately it's not possible to create an account for injected web3, please use the Metamask or other providers"
```

Comment:
typo `Unfortunately`

Reply:
little rewording: 
`Unfortunately it's not possible to create an account using injected web3. Please create the account directly from your provider (i.e metamask or other of the same type).`